.github/workflows/maven.yml

```diff
@@ -31,6 +31,15 @@ jobs:
         maven-version: 3.9.6
     - name: Build with Maven
       run: mvn -B package --file pom.xml -DunofficialSuffix=
+    - name: Arrange P2 Repsository
+      run: |
+        cd com.wamas.ide.launching.update/target/repository/
+        mv plugins/* .
+        mv features/* .
+        sed -i 's/\/plugins\//\//g' artifacts.xml
+        sed -i 's/\/binary\//\//g' artifacts.xml
+        sed -i 's/\/features\//\//g' artifacts.xml
+        rm -r plugins features
```

Comment:
Since we expect `*.jar` only, this should fail when something else is inside these directories, given that we move `*.jar` above.